ldk/javascript/src/whisper/mapper.ts

```diff
@@ -49,8 +50,13 @@ export function mapToInternalChildComponent(
           : undefined,
       } as WhisperService.Box;
     case WhisperComponentType.Breadcrumbs:
+      // eslint-disable-next-line no-case-declarations
+      const { links, ...props } = component as Breadcrumbs;
```

Comment:
`const { links, ...props } = component as WhisperComponentType.Breadcrumbs` Maybe this is better? So we don't need to import Breadcrumb. 0.0

Reply:
Responded on other comment